src/pages/docs/platform/architecture/infrastructure-operations.mdx

```diff
@@ -1,10 +1,9 @@
 ---
 title: Infrastructure operations
+intro: "Infrastructure operations cover the people, systems, and processes that pertain to operating a distributed system. They are an essential complement to the technical architecture of a distributed system; the operations and the architecture combine to achieve the desired properties of the system such as scalability and availability."
```

Comment:
I'm not really sure what point this text is getting at here...

Let's re-think this one